src/04-modality-specific-files/08-genetic-descriptor.md

```diff
@@ -0,0 +1,60 @@
+# Genetic Descriptor
+
+The extension was primarily developped by Cyril Pernet and Clara Moreau with contributions from Tom Nichols and Jessica Turner.
+
+The goal of the genetic descriptor is to link imaging and genetic data by providing minimal information about i) where to find genetic information associated with the imaging data and ii) what type of genetic information is available. This is necessary as genetic data are typically stored in dedicated repositories, separately from the imaging data.
+
+## dataset_description.json
+
+Two additional keys related to the genetic data can be added. The Key `GeneticDataBase` (MANDATORY) links to the name of the database and web address. The key `GeneticDescriptor` (OPTIONAL) refers to the descriptor (e.g. journal article) of the genetic data.
+
+`dataset_description.json` example:
+
+{  
+  "Name": "xxxxx",  
+  "BIDSVersion":  "xxxxx",  
+  "License": "CC0",  
+  "Authors": ["1st author", "2nd author"],  
+  "Funding": "list your funding sources",  
+  "GeneticDataBase": ["Name", "www",],  
+  "GeneticDescriptor": ["data paper", "10.0.2.3/dfjj.10"]  
+}
+
+## sub- and participant.tsv
```

Comment:
Perhaps considering to change the header to follow some of our current formating. [an example location of my thoughts](https://bids-specification.readthedocs.io/en/latest/03-modality-agnostic-files.html#participants-file)

```suggestion
## Subject naming and Participants file
```